libevmasm/PathGasMeter.h

```diff
@@ -57,6 +57,11 @@ class PathGasMeter
 
 	GasMeter::GasConsumption estimateMax(size_t _startIndex, std::shared_ptr<KnownState> const& _state);
 
+	static GasMeter::GasConsumption estimateMax(AssemblyItems const& _items, solidity::EVMVersion _evmVersion, size_t _startIndex, std::shared_ptr<KnownState> const& _state)
```

Comment:
Code formatting rule here?

Reply:
You could write
```
static GasMeter::GasConsumption estimateMax(
    AssemblyItems const& _items,
    solidity::EVMVersion _evmVersion,
    size_t _startIndex,
    std::shared_ptr<KnownState> const& _state
)
```
Or did you mean something else?

Reply:
Yep, wasn't sure what's the rule.